internal/codegen/golang/query.go

```diff
@@ -128,14 +128,16 @@ func (v QueryValue) Params() string {
 	}
 	var out []string
 	if v.Struct == nil {
-		if !v.Column.IsSqlcSlice && strings.HasPrefix(v.Typ, "[]") && v.Typ != "[]byte" && !v.SQLDriver.IsPGX() {
+		if v.Column.IsSqlcDynamic {
+		} else if !v.Column.IsSqlcSlice && strings.HasPrefix(v.Typ, "[]") && v.Typ != "[]byte" && !v.SQLDriver.IsPGX() {
 			out = append(out, "pq.Array("+escape(v.Name)+")")
 		} else {
 			out = append(out, escape(v.Name))
 		}
 	} else {
 		for _, f := range v.Struct.Fields {
-			if !f.HasSqlcSlice() && strings.HasPrefix(f.Type, "[]") && f.Type != "[]byte" && !v.SQLDriver.IsPGX() {
+			if f.HasSqlcDynamic() {
```

Comment:
Hi @ovadbar there is no logic inside this condition. May be it is not required?

Reply:
Maybe I have should have written it as the following.  The code is needed
```
if !f.HasSqlcDynamic() {
    if !f.HasSqlcSlice() && strings.HasPrefix(f.Type, "[]") && f.Type != "[]byte" && !v.SQLDriver.IsPGX() {
	out = append(out, "pq.Array("+escape(v.VariableForField(f))+")")
    } else {
	out = append(out, escape(v.VariableForField(f)))
    }
}
```